docroot/themes/custom/mappy/mappy.theme

```diff
@@ -4,3 +4,13 @@
  * @file
  * Functions to support theming in the Mappy theme.
  */
+
+function mappy_page_attachments_alter(&$page) {
+  $current_path = \Drupal::service('path.current')->getPath();
```

Comment:
This seems reasonable, I'm assuming you looked in `$page` and the nid was nowhere to be found there?